src/hooks/v2/contractReader/NftCIDsOf.ts

```diff
@@ -2,8 +2,8 @@ import { V2ContractName } from 'models/v2/contracts'
 
 import useV2ContractReader from './V2ContractReader'
 
-export function useNftCidOf(projectId: number | undefined) {
-  return useV2ContractReader<string>({
+export function useNftCIDsOf(projectId: number | undefined) {
```

Comment:
Aren't we doing this casing now? :P

```suggestion
export function useNftCidsOf(projectId: number | undefined) {
```

Reply:
I thought since `CID` is plural in these cases (no pun intended) it would be more readable to go with `CIDs`

Reply:
yea tbh i tend to like full caps for acronyms. sgtm